config/environments/production.rb

```diff
@@ -74,6 +79,16 @@
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
 
+  # Use a different logger for distributed setups.
+  # require 'syslog/logger'
+  config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new(ENV['app_name'] || 'rest-whois'))
```

Comment:
I'd Move this whole block under line 57, I think it's quite weird to have the file organized in a way:

[logging]
[mailer]
[logging again]